src/main/java/ch/njol/skript/classes/data/DefaultChangers.java

```diff
@@ -299,7 +302,16 @@ public void change(final Block[] blocks, final @Nullable Object[] delta, final C
 						assert delta != null;
 						Object o = delta[0];
 						if (o instanceof ItemType) {
-							((ItemType) delta[0]).getBlock().setBlock(block, true);
+							ItemType itemType = (ItemType) o;
+							itemType.getBlock().setBlock(block, true);
+							if (itemType.getItemMeta() instanceof SkullMeta) {
+								OfflinePlayer offlinePlayer = ((SkullMeta) itemType.getItemMeta()).getOwningPlayer();
+								if (offlinePlayer == null)
+									return;
```

Comment:
We don't want to stop all other blocks as well
```suggestion
									continue;
```

Reply:
oh oops